torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -97,11 +133,61 @@ void IterVisitor::traverseFrom(
   }
 }
 
+// This is a simplified version of traverseFrom
+void IterVisitor::traverseFrom2(
+    Fusion* const fusion,
```

Comment:
nit: the const is not really needed here

Reply:
I didn't want to change the original signature.

Reply:
LGTM you can replace the original if you think this is cleaner.